src/compiler/types.ts

```diff
@@ -10013,7 +10013,7 @@ export const enum ListFormat {
     SourceFileStatements = MultiLine | NoTrailingNewLine,
     Decorators = MultiLine | Optional | SpaceAfterList,
     TypeArguments = CommaDelimited | SpaceBetweenSiblings | SingleLine | AngleBrackets | Optional,
-    TypeParameters = CommaDelimited | SpaceBetweenSiblings | SingleLine | AngleBrackets | Optional,
+    TypeParameters = CommaDelimited | SpaceBetweenSiblings | SingleLine | AngleBrackets | Optional | AllowTrailingComma,

```

Comment:
@weswigham is it acceptable to allow a trailing comma in all type parameters if it exists, or is it better to exclude it in certain cases?

Reply:
https://github.com/microsoft/TypeScript/pull/59612 - definitely only want it for arrow functions. The trailing comma looks odd everywhere, but is only syntactically required for some arrows.

Reply:
@weswigham ok